timemachine/fe/free_energy.py

```diff
@@ -926,9 +933,16 @@ def log_q_fn(replica_idx: ReplicaIdx, state_idx: StateIdx) -> float:
         # Add an identity move to the mixture to ensure aperiodicity
         neighbor_pairs = [(StateIdx(0), StateIdx(0))] + neighbor_pairs
 
+    # Setup the barostat to run more often for equilibration
+    equil_barostat_interval = 15
+    barostat = context.get_barostat()
+    if barostat is not None:
+        barostat.set_interval(equil_barostat_interval)
+
     def get_equilibrated_xvb(xvb: CoordsVelBox, params: NDArray) -> CoordsVelBox:
         if md_params.n_eq_steps == 0:
             return xvb
+        # Set the movers to 0 to ensure they all equilibrate the same way
```

Comment:
nit: was this intentional? (redundant comment)

Reply:
Nope, will remove

Reply:
Removed in https://github.com/proteneer/timemachine/pull/1221/commits/5942f55f2bbbbb20a8dfc92f862516381e2dd570